mmv1/products/redis/Cluster.yaml

```diff
@@ -378,6 +381,14 @@ properties:
         type: String
         description: |
           Immutable. The zone for single zone Memorystore Redis cluster.
+  - name: 'allowFewerZonesDeployment'
+    type: Boolean
+    description: |
+      Allows customers to specify if they are okay with deploying a multi-zone
+      cluster in less than 3 zones. Once set, if there is a zonal outage during
+      the cluster creation, the cluster will only be deployed in 2 zones, and
+      stay within the 2 zones for its lifecycle.
+    default_value: false
```

Comment:
Is there reason we don't set to true in the test. We are setting to a value different from API value. Does api return false by default?

Reply:
Good point @ScottSuarez Ill update and set to true